apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/stu3/providers/HHAClaimTransformerTest.java

```diff
@@ -41,14 +43,15 @@ public void transformSampleARecord() throws FHIRException {
             .findFirst()
             .get();
 
-    ExplanationOfBenefit eob =
-        HHAClaimTransformer.transform(
-            new TransformerContext(
-                new MetricRegistry(),
-                Optional.empty(),
-                FdaDrugCodeDisplayLookup.createDrugCodeLookupForTesting()),
-            claim);
-    assertMatches(claim, eob);
+    TransformerContext transformerContext =
+        new TransformerContext(
+            new MetricRegistry(),
+            Optional.empty(),
+            FdaDrugCodeDisplayLookup.createDrugCodeLookupForTesting(),
+            NPIOrgLookup.createNpiOrgLookupForTesting());
```

Comment:
I thought this change was for v2 only; why are we adjusting the v1 transformer tests?

Reply:
Changed for v1 as well since it would be reading from the static file. The mapping stays the same